doc/man/cgget.1

```diff
@@ -55,6 +55,11 @@ This option can be used multiple times.
 .B -v, --values-only
 print only values, not parameter names.
 
+.TP
+.B -m
```

Comment:
Should this be after `-h` and before `-n`?  It appears the list was alphabetical before this change.

Reply:
@drakenclimber Thank you so much for the review comments. Agreed on all the comments, will re-work and push the v2.